server/src/core/util/validation-util.ts

```diff
@@ -19,3 +19,43 @@ export const idValidationSchema = joi.string()
   });
 
 export const validateId = makeValidatorFromJoiSchema(idValidationSchema);
+
+export const phoneValidationSchema = joi.string()
+  .required()
+  .pattern(/^2547\d{8}$/) // Starts with 2547 and ends with 8 digits
+  .messages({
+    'any.required': 'Phone is required',
+    'string.base': 'Invalid type, phone must be a string',
+    'string.empty': 'Please enter your phone number',
+    'string.pattern.base': 'Invalid phone number. Must start with 2547 and be 12 digit long'
+  });
+
+export const passwordValidationSchema = joi.string()
+  .required()
+  .pattern(/^.{8,18}$/)
+  .messages({
+    'any.required': 'Password is required',
+    'string.base': 'Invalid type, password must be a string',
+    'string.empty': 'Please enter your password',
+    'string.pattern.base': 'Invalid password. Must range between 8 and 18 characters'
+});
+
+export const googleIdTokenValidationSchema = joi.string()
```

Comment:
Again, why is it a requirement to provide a google id token especially when a phone number and password have already been provided?